test/view_component/integration_test.rb

```diff
@@ -280,4 +280,31 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     assert_select("p", text: "Radio clock counter: 1")
     assert_select("p", text: "Mints counter: 2")
   end
+
+  if Rails.version.to_f < 6.1
+    test "does not include render monkey patches if render_monkey_patch_enabled config is set to false" do
+      begin
+        old_value = Rails.application.config.view_component.render_monkey_patch_enabled
+
+        Rails.application.config.view_component.render_monkey_patch_enabled = false
```

Comment:
Setting this option here to false does not work since the initializers of `ViewComponent::Engine` only run once when the rails app gets booted.  Still not sure how to solve this.

Reply:
Resolving this since the code moved to a different location. Will add this comment to the new location.